src/DrHouse/HealthData.cs

```diff
@@ -14,13 +16,18 @@ public HealthData(string appName)
             DependenciesStatus = new List<HealthData>();
         }
 
+        [DataMember(EmitDefaultValue = false)]
         public string Name { get; private set; }
 
+        [DataMember(EmitDefaultValue = false)]
         public string Type { get; set; }
 
+        [DataMember(EmitDefaultValue = false)]
         public string ErrorMessage { get; set; }
-
+        
         private bool _isOk;
+
+        [DataMember(EmitDefaultValue = false)]
```

Comment:
This property should always be serialized. Did you check what would happen if ```IsOK``` is ```false``` and ```EmitDefaultValue = false```?

Reply:
You're right. My intention was to hide any default param value but at this time check if the state is ok or not is a core for his.